Sources/UI/Extensions/UIKit/UIViewController/UIViewController+ImagePicker.swift

```diff
@@ -70,7 +70,17 @@ extension UIViewController {
         present(imagePickerViewController, animated: true)
     }
     
-    func showImpagePickerAuthorizationStatusAlert(_ status: PHAuthorizationStatus) {
+    private func showImagePickerAlert(title: String, message: String) {
```

Comment:
I'm not sure that we should show the alert in this case.

Reply:
You mean if the file size exceeds the limit?

Reply:
Yes. How a customer can handle that in own way?

Reply:
Well they don't have the ability to handle status alert in their own way too...

Reply:
I think will be better if you will change it to  use`ViewController:show(errorMessage:)` then customers could override it.

Reply:
That makes sense 👍 Should I do the same for status alerts? `show(errorMessage:)` does not accept a title/message though